insertSizeMetrics.wdl

```diff
@@ -0,0 +1,84 @@
+version 1.0
+
+workflow insertSizeMetrics {
+  input {
+    File inputBam
+    String outputFileNamePrefix = basename(inputBam, '.bam')
```

Comment:
If variable has a default, it is optional.

Reply:
It is not very obvious here, according to wd docs '?' means that the user does not have to specify the input. Since you don't have '?' here it suggests that the user does need to provide all these parameters. I find it confusing.